src/lib/useRoller.ts

```diff
@@ -161,69 +155,76 @@ export default function useRoller() {
         new Set<number>()
       );
 
-      console.log(pendingTxs, pendingSpawns);
-
       planets = planets.filter((point: number) => !pendingSpawns.has(point));
 
-      const starInfo = nonces[_point];
-      console.log(starInfo);
+      const starInfo = await api.getPoint(_point);
       const tickets: {
         ticket: string;
         planet: number;
         owner: string;
       }[] = [];
       const requests: Promise<string>[] = [];
 
-      const operator =
-        getSpawnNonce(starInfo, _wallet.address) ||
-        getOwnerNonce(starInfo, _wallet.address);
+      const proxy = isOwnerProxy(starInfo.ownership!, _wallet.address)
+        ? 'own'
+        : isSpawnProxy(starInfo.ownership!, _wallet.address)
+        ? 'spawn'
+        : undefined;
 
-      if (operator === undefined)
+      if (proxy === undefined)
         throw new Error("Error: Address doesn't match proxy");
-      console.log('operator', operator);
+
+      const nonce = await api.getNonce({
```

Comment:
I think this is a great design decision to push the nonce management to the backend. simplifies the frontend significantly